### PR TITLE
Missing Changelog items

### PR DESCRIPTION
**Pull request by maackle**
### Summary

Forgot to add changelog items for #2156, here they are.

### TODO:
- [x] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

**Comment by ThetaSinner** [comment]
Thanks for picking up my mistake with the storage info changes 👍

File: crates/holochain/CHANGELOG.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Adds new functionality to the conductor admin API which returns disk storage information. The storage used by apps is broken down into blobs which are being used by one or more app.
```
Comment (jost-s) [inline crates/holochain/CHANGELOG.md:10]: Why are you adding this to the main changelog directly? It'll make it there under the `holochain_conductor_api` heading if it's that crate's changelog.

Comment (jost-s) [inline crates/holochain/CHANGELOG.md:10]: Personally I'd write this _either_ in one _or_ another changelog.

**Review by jost-s — APPROVED** [review]

File: crates/holochain/CHANGELOG.md (hunk 1 above)
Comment (maackle) [reply crates/holochain/CHANGELOG.md]: Agreed, I removed it here

**Pull request merged**